app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,3 @@
 @charset "utf-8";
 
-@import "normalize.css/normalize.css";
+// @import "normalize.css/normalize.css";
```

Comment:
Having a normalize stylesheet is usually a good thing. But really, that's up to you and how you wish to structure your stylesheets :)

Reply:
Thanks Elle, I was getting an error that it couldn't load it, so that was a question I had for you when I see you next.

The authentication is hopefully going to be both oauth and bcrypt but it will be implemented through a sessions controller which will handle login and logout and setting the session hash.